config/environment.rb

```diff
@@ -4,3 +4,15 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+# Set up ActionMailer for SMTP
+
+ActionMailer::Base.smtp_settings = {
+  :address        => ENV["SMTP_SERVER"],
+  :domain         => ENV["EMAIL_DOMAIN"],
+  :port           => 587,
```

Comment:
Maybe this, `:authentication`, and `:enable_starttls_auto` should be optional environment variables so other users can overwrite them easily?

That can be done easily here using `.blank?` ([here's the rails documentation](http://api.rubyonrails.org/v4.2/classes/Object.html#method-i-blank-3F)). e.g.

````ruby
ENV["SMTP_PORT"].blank? ? ENV["SMTP_PORT"].to_i : DEFAULT_SMTP_PORT
````

(and `DEFAULT_SMTP_PORT` can be defined just above this call to avoid the [magic number](http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)).